### DISCUSSION TITLE
Playbook is not running

### DISCUSSION THREAD

**Discussion by guddu07** [alivx/CIS-Ubuntu-20.04-Ansible#62] [General]
Hi,

I am running this playbook in Ubuntu 20.04 EC2 instance and getting this error at very first stage. I even changed gather_facts to yes but still same issue. If anybody can please help me on this.

    TASK [CIS-Ubuntu-20.04-Ansible : Preflight - Fail if host is not suitable for this benchmark] *****************************************************************
    fatal: [localhost]: FAILED! => {"changed": false, "msg": "This benchmark is not suitable for the destination operating system"}

**Comment by guddu07** [comment]
I managed to run after disabling preflight.yaml however further I am getting this error. If anybody can please help. I had run all_mounts.py script before but still I keep getting this error 

    TASK [CIS-Ubuntu-20.04-Ansible : 1.1.8 - Ensure nodev option set on /var/tmp partition] ***********************************************************************
    fatal: [localhost]: FAILED! => {"msg": "'ansible_mounts' is undefined"}

**Reply by alivx** [reply]
can you share the all_mounts.py  or share the output of that script so I can check
